backend/app/utils/memory_curator.py

```
@@ -1004,6 +1005,13 @@ def memory_curator_run(
         except Exception:
             logger.error("Failed to drop stale memories", exc_info=True)
 
+        try:
+            emptied = db_prune_empty_memories(preferences.min_images)
+            if emptied:
+                logger.info(f"Marked {emptied} memories empty (images/folders removed)")
+        except Exception:
+            logger.error("Failed to prune empty memories", exc_info=True)
```

Ruff (0.16.1)

[warning] Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)